broker/core/src/processing/feeder.cc

```diff
@@ -95,6 +95,7 @@ feeder::feeder(const std::string& name,
  *  Destructor.
  */
 feeder::~feeder() {
+  stop();
```

Comment:
if engine_ptr is null, it is because, the engine is already switched off. Since this patch, I have no more issue with it.

Reply:
The applier::endpoint, when stopped, delete all the failovers and feeders. Failovers are well stopped but not feeders (now). So if we add this stop in the destructor, it is really stopped because we wait for that. And then when the engine is stopped and destroyed there no more feeders or failovers that could crash.